tests/Core/Components/Link/FluentLinkTests.razor

```diff
@@ -0,0 +1,224 @@
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink hreflang="en">fluent-link</FluentLink>);
+
+		// Assert
+		cut.Verify();
+	}
+
+	[Theory]
+	[InlineData(LinkReferrerPolicy.NoReferrer, "no-referrer")]
+	[InlineData(LinkReferrerPolicy.NoReferrerWhenDowngrade, "no-referrer-when-downgrade")]
+	[InlineData(LinkReferrerPolicy.Origin, "origin")]
+	[InlineData(LinkReferrerPolicy.OriginWhenCrossOrigin, "origin-when-cross-origin")]
+	[InlineData(LinkReferrerPolicy.UnsafeUrl, "unsafe-url")]
+	[InlineData((LinkReferrerPolicy)999, "")]
+	public void FluentLink_ReferrerPolicy(LinkReferrerPolicy referrerPolicy, string expectedAttribute)
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink ReferrerPolicy="@referrerPolicy">fluent link</FluentLink>);
+
+		// Assert
+		cut.MarkupMatches($"<fluent-link referrerPolicy=\"{expectedAttribute}\">fluent link</fluent-link>");
+	}
+
+	[Theory]
+	[InlineData(LinkRel.Alternate, "alternate")]
+	[InlineData(LinkRel.Author, "author")]
+	[InlineData(LinkRel.Canonical, "canonical")]
+	[InlineData(LinkRel.Expect, "expect")]
+	[InlineData(LinkRel.Help, "help")]
+	[InlineData(LinkRel.License, "license")]
+	[InlineData(LinkRel.Manifest, "manifest")]
+	[InlineData(LinkRel.Me, "me")]
+	[InlineData(LinkRel.Next, "next")]
+	[InlineData(LinkRel.Prev, "prev")]
+	[InlineData(LinkRel.PrivacyPolicy, "privacy-policy")]
+	[InlineData(LinkRel.Search, "search")]
+	[InlineData(LinkRel.TermsOfService, "terms-of-service")]
+	[InlineData((LinkRel)999, "")]
+	public void FluentLink_Rel(LinkRel rel, string expectedAttribute)
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink Rel="@rel">fluent link</FluentLink>);
+
+		// Assert
+		cut.MarkupMatches($"<fluent-link rel=\"{expectedAttribute}\">fluent link</fluent-link>");
+	}
+
+	[Fact]
+	public void FluentLink_Type()
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink LinkType="text/css">fluent-link</FluentLink>);
+
+		// Assert
+		cut.Verify();
+	}
+
+
+	[Theory]
+	[InlineData(LinkAppearance.Default)]
+	[InlineData(LinkAppearance.Subtle)]
+	[InlineData((LinkAppearance)999)]
+	public void FluentLink_AppearanceAttribute(LinkAppearance appearance)
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink Appearance="@appearance">fluent-link</FluentLink>);
+
+		// Assert
+		var actual = cut.Find("fluent-link").GetAttribute("appearance");
+		Assert.Equal(appearance.ToAttributeValue(returnEmptyAsNull: true), actual);
+	}
+
+	[Fact]
+	public void FluentLink_ClassAttribute()
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink Class="additional-class">fluent-link</FluentLink>);
+
+		// Assert
+		cut.Verify();
+	}
+
+	[Fact]
+	public void FluentLink_StyleAttribute()
+	{
+		// Arrange && Act
+		var cut = Render(@<FluentLink Style="background-color: green;">fluent-link</FluentLink>);
+
+		// Assert
+		cut.Verify();
+	}
+
+	[Theory]
+	[InlineData("id-value")]
+	[InlineData(null)]
+	[InlineData("")]
+	[InlineData(" ")]
+	public void FluentLink_IdAttribute(string? id)
```

Comment:
You can keep these tests, but for info, the "common" attributes are automatically validated by the `ComponentBaseTests.cs` tests: Id, Class, Style, Margin, Padding, extra-attribute, ... :-)